### PR TITLE
docs: i18n

### PR DESCRIPTION
**Pull request by hacdias**
@lidel I stole some of your words on https://github.com/ipfs-shipyard/ipfs-webui/pull/772!

Could you take a look and add/update anything if you feel to?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +120,32 @@ To do a single run of the tests and generate a coverage report, run the followin
 npm run test:coverage
 ```
 
+### i18n
+
+The translations are stored on [./public/locales](./public/locales) and the English version is the source of truth.
+
+We use Transifex to help us translate WebUI to another languages. If you're interesting in contributing, go to [our page on Transifex](https://www.transifex.com/ipfs/ipfs-webui/translate/), create an account, pick up a language and start translating.
```
Comment (lidel) [inline README.md]: `interesting` → `interested`

File: README.md [hunk 2]
```diff
@@ -120,6 +120,32 @@ To do a single run of the tests and generate a coverage report, run the followin
...
+
+#### To Start Translating
+
+1. [Create Transifex account](https://www.transifex.com/signup/?join_project=ipfs-webui) 
+2. Go to https://www.transifex.com/ipfs/ipfs-webui/translate/, pick up the language, and start translating
```
Comment (lidel) [inline README.md]: `pick up the language` → `pick a language`

File: README.md (hunk 1 above)
Comment (hacdias) [reply README.md]: 😆

**Comment by hacdias** [comment]
Done and merged @lidel

**Pull request merged**